turingschool-examples/romcom
# Feature/create random cover

## What is the change?
FEATURE: When you click on Create Random Cover, a new cover image, title and two descriptors are randomly generated and displayed.

## Where should the reviewer start?
Line 30 - the function generateRandomCover.

## How should this be tested?
Open index.html, click on the Create Random Cover button and a random cover should be displayed.
